canopen/profiles/p402.py

```diff
@@ -60,12 +89,26 @@ def setup_402_state_machine(self):
         # Use register as to stay manufacturer agnostic
         self.pdo.tx[1].add_variable(0x6041)
         # add callback to listen to TPDO1 and change 402 state
+        self.pdo.tx[1].clear_callbacks()
         self.pdo.tx[1].add_callback(self.powerstate_402.on_PDO1_callback)
         self.pdo.tx[1].trans_type = 255
         self.pdo.tx[1].enabled = True
         self.pdo.tx[1].save()
         self.nmt.state = 'OPERATIONAL'
 
+    def add_callback_402_state(self, state, cb):
+        self.callbacks_402[state].append(cb)
+
+    def clear_402_callbacks(self):
+        print "clearing 402 callbacks"
```

Comment:
print statement only works on Python 2. Either change to print(), remove it if not needed, or use logging module instead.